lib/puppet_metadata/operatingsystem.rb

```diff
@@ -13,10 +13,10 @@ class OperatingSystem
     # The EOL dates for the various operating systems
     # @see .eol_date
     EOL_DATES = {
-      # https://almalinux.org/ FAQ at the bottom
+      # https://wiki.almalinux.org/release-notes/
       'AlmaLinux' => {
-        '9' => nil,
-        '8' => '2029-05-31',
+        '9' => '2027-05-31',
+        '8' => '2024-05-31',
```

Comment:
It will remain supported:
> Version 8 will have active support until 31 May 2024, and security support until 31 May 2029.
```suggestion
        '8' => '2029-05-31',
```